windows/runner/flutter_window.cpp

```diff
@@ -26,6 +27,47 @@ bool FlutterWindow::OnCreate() {
   }
   RegisterPlugins(flutter_controller_->engine());
   SetChildContent(flutter_controller_->view()->GetNativeWindow());
+
+  // this is for notifying the Dart code of a closing event.
+  notificationChannel = std::make_unique<flutter::MethodChannel<flutter::EncodableValue>>(
+      flutter_controller_->engine()->messenger(),
+      "splash_window_close_notification",
+      &flutter::StandardMethodCodec::GetInstance());
+
+  // this is to react to the Dart code invocations.
+  auto methodChannel = std::make_unique<flutter::MethodChannel<flutter::EncodableValue>>(
+      flutter_controller_->engine()->messenger(),
+      "splash_window_close",
+      &flutter::StandardMethodCodec::GetInstance());
```

Comment:
i learned recently in https://github.com/invertase/flutter_desktop_webview_auth/pull/4 that it's possible to use one channel both ways ;)

Reply:
Indeed! I liked that! Thanks for the tip.